src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
Yeah, saves a few characters, but it's not worth losing the clarity of `<a-event name="XXX">` (tells you that it is an event handler, and gives the exact event name). Anyways, we can remove next release.